rest/pom.xml

```diff
@@ -35,6 +37,24 @@
         <rest.models.package>${root.package}.rest.models</rest.models.package>
     </properties>
 
+    <build>
+        <resources>
+            <!--
+                Add any OWASP suppression file (must be named owasp-suppressions.xml) to the JARs
+                created. This will allow projects using those JARs as dependencies to include the
+                file in their OWASP plugin configuration and not have to repeat and maintain the
+                same suppressions.
+            -->
+            <resource>
+                <filtering>false</filtering>
+                <directory>${basedir}</directory>
+                <includes>
+                    <include>owasp-suppressions.xml</include>
```

Comment:
We currently have a false positive for `Vulnerabilities related to Gradle, not springfox`. Would this be a real vulnerability if a gradle project depended on com.connexta.ingest/ingest-api-rest-spring-stubs? That project shouldn't pull in that suppression, right?